src/test/java/com/airlines/dao/impl/CrewDaoImplTest.java

```diff
@@ -0,0 +1,227 @@
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThrows;
+import static org.junit.Assert.assertTrue;
+import static com.airlines.model.Citizenship.BRAZILIAN;
+import static com.airlines.model.Citizenship.ITALIAN;
+import static com.airlines.model.Position.PILOT_IN_COMMAND;
+
+public class CrewDaoImplTest {
+    private static final String SCHEMA_SQL_PATH = "src/test/resources/sql/schema.sql";
+    private static final String DATA_SQL_PATH = "src/test/resources/sql/test_data.sql";
+    private static final String CREATE_TABLES_QUERY = JdbcUtil.getSqlQueryString(SCHEMA_SQL_PATH);
+    private static final String GENERATE_DATA_QUERY = JdbcUtil.getSqlQueryString(DATA_SQL_PATH);
+
+    private static final CrewMember CREW_MEMBER = buildTestCrewMember();
+    private static final Crew CREW = buildCrewFromDataSet();
+
+    private static CrewDao crewDao;
+
+    private static DataSource dataSource;
+
+    @BeforeClass
+    public static void init() {
+        dataSource = TestDatasourceProvider.createTestDatasource();
+        crewDao = new CrewDaoImpl(dataSource);
+    }
+
+    @Before
+    public void dataBaseInit() throws SQLException {
+        createTables();
+    }
+
+    @Test
+    public void shouldThrowExceptionIfCrewMemberWithNoIdProvidedToAddMethod() {
+        CrewMember crewMember = CrewMember.builder()
+                .withId(null)
+                .build();
+
+        IllegalArgumentException exception = assertThrows(IllegalArgumentException.class, () -> crewDao.add(crewMember, null));
+
+        assertEquals("Valid crewMember entity should be provided", exception.getMessage());
+    }
+
+    @Test
+    public void shouldThrowExceptionIfCrewProvidedWithoutIdToAddMethod() {
+        Crew crew = new Crew(null, "Some name", Collections.singletonList(CREW_MEMBER));
+
+        IllegalArgumentException exception = assertThrows(IllegalArgumentException.class, () -> crewDao.add(CREW_MEMBER, crew));
+
+        assertEquals("Valid crew entity should be provided", exception.getMessage());
+    }
+
+    @Test
+    public void shouldCorrectlyAddCrewMemberToCrew() throws SQLException {
+        populateTestData();
+        crewDao.add(CREW_MEMBER, CREW);
+        List<CrewMember> list = crewDao.getByCrewId(CREW.getId());
+        assertTrue(list.contains(CREW_MEMBER));
+    }
+
+    @Test
+    public void shouldGetCrewMembersByCrewId() throws SQLException {
+        populateTestData();
+        List<CrewMember> list = crewDao.getByCrewId(1L);
+        assertEquals(3, list.size());
+
+        CrewMember crewMember1 = list.get(0);
+        CrewMember crewMember2 = list.get(1);
+        CrewMember crewMember3 = list.get(2);
+
+        assertEquals("Igor", crewMember1.getFirstName());
+        assertEquals("Shpack", crewMember1.getLastName());
```

Comment:
firstName could be same for different persons, make two assertions:
check Id
check LastName

In general case, we should even check all fields, but in this particular case, we could just try compare two most unique values